AMW_business/src/main/java/ch/puzzle/itc/mobiliar/business/apps/control/AppsService.java

```diff
@@ -0,0 +1,121 @@
+package ch.puzzle.itc.mobiliar.business.apps.control;
+
+import ch.puzzle.itc.mobiliar.business.apps.boundary.*;
+import ch.puzzle.itc.mobiliar.business.foreignable.entity.ForeignableOwner;
+import ch.puzzle.itc.mobiliar.business.releasing.boundary.ReleaseLocator;
+import ch.puzzle.itc.mobiliar.business.releasing.entity.ReleaseEntity;
+import ch.puzzle.itc.mobiliar.business.resourcegroup.boundary.ResourceBoundary;
+import ch.puzzle.itc.mobiliar.business.resourcegroup.boundary.ResourceRelations;
+import ch.puzzle.itc.mobiliar.business.resourcegroup.control.ResourceTypeRepository;
+import ch.puzzle.itc.mobiliar.business.resourcegroup.entity.Application;
+import ch.puzzle.itc.mobiliar.business.resourcegroup.entity.Resource;
+import ch.puzzle.itc.mobiliar.business.resourcegroup.entity.ResourceTypeEntity;
+import ch.puzzle.itc.mobiliar.business.resourcegroup.entity.ResourceWithRelations;
+import ch.puzzle.itc.mobiliar.business.security.boundary.PermissionBoundary;
+import ch.puzzle.itc.mobiliar.business.security.entity.Permission;
+import ch.puzzle.itc.mobiliar.business.security.interceptor.HasPermission;
+import ch.puzzle.itc.mobiliar.common.exception.AMWException;
+import ch.puzzle.itc.mobiliar.common.exception.ElementAlreadyExistsException;
+import ch.puzzle.itc.mobiliar.common.exception.NotFoundException;
+import ch.puzzle.itc.mobiliar.common.exception.ResourceTypeNotFoundException;
+import ch.puzzle.itc.mobiliar.common.util.DefaultResourceTypeDefinition;
+import ch.puzzle.itc.mobiliar.common.util.Tuple;
+
+import javax.ejb.Stateless;
+import javax.inject.Inject;
+import java.util.List;
+
+import static ch.puzzle.itc.mobiliar.business.security.entity.Action.CREATE;
+
+@Stateless
+public class AppsService implements ListAppsUseCase, AddAppServerUseCase, AddAppUseCase, AddAppWithServerUseCase {
```

Comment:
I'm not sure if I really like it that this Service implements all use cases. Maybe its ok. I have to think about it...